fighthealthinsurance/views.py

```diff
@@ -163,31 +171,28 @@ def get(self, request):
             "remove_data.html",
             context={
                 "title": "Remove My Data",
-                "form": DeleteDataForm(),
+                "form": core_forms.DeleteDataForm(),
             },
         )
 
     def post(self, request):
-        form = DeleteDataForm(request.POST)
+        form = core_forms.DeleteDataForm(request.POST)
+
         if form.is_valid():
             email = form.cleaned_data["email"]
-            RemoveDataHelper.remove_data_for_email(email)
```

Comment:
Since this **if** returns, just moved the else outside of an else to avoid awkward branching

Reply:
love it